apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -0,0 +1,584 @@
+                                          `
+                                        : ''}
+                                      <button
+                                        class="btn btn-sm btn-outline-primary"
+                                        type="button"
+                                        data-toggle="modal"
+                                        data-target="#publiclyShareModal"
+                                      >
+                                        Share Publicly
+                                      </button>
+                                      ${publiclyShareModal({
+                                        isEnterprise,
+                                        csrfToken: resLocals.__csrf_token,
+                                        qid: resLocals.question.qid,
+                                      })}
+                                    `
+                                  : ''}
+                              `}
+                        </td>
+                      </tr>
+                    `
+                  : ''}
+                ${resLocals.question.type === 'Freeform' &&
+                resLocals.question.grading_method !== 'External' &&
+                resLocals.authz_data.has_course_permission_view
+                  ? html`
+                      <tr>
+                        <th class="align-middle">Tests</th>
+                        <td>
+                          <form
+                            name="question-tests-form"
+                            method="POST"
+                            action="${questionTestPath}"
+                          >
+                            <input
+                              type="hidden"
+                              name="__csrf_token"
+                              value="${questionTestCsrfToken}"
+                            />
+                            <button
+                              class="btn btn-sm btn-outline-primary"
+                              name="__action"
+                              value="test_once"
+                            >
+                              Test once with full details
+                            </button>
+                            <button
+                              class="btn btn-sm btn-outline-primary"
+                              name="__action"
+                              value="test_100"
+                            >
+                              Test 100 times with only results
+                            </button>
+                          </form>
+                        </td>
+                      </tr>
+                    `
+                  : ''}
+              </tbody>
+            </table>
+
+            <!--
+          We only want to show the card footer if it will have something inside. Since it is easy
+          to make a mistake when merging logical conditions, we will simply "or" them all.
+        -->
```

Comment:
Is this comment necessary?

Reply:
Removed in [e8cc26c](https://github.com/PrairieLearn/PrairieLearn/pull/9571/commits/e8cc26c79af51815e4e9bdbf5d82f9f432f5decc).